standalone/src/widget/index.tsx

```diff
@@ -73,7 +75,14 @@ async function initWidget(store: Store) {
 }
 
 async function initStoreWidget(store: Store, channelID: string) {
-    await store.dispatch(getChannelAction(channelID));
+    // initialize some basic state.
+    await Promise.all([
+        getMe()(store.dispatch, store.getState),
+        getMyPreferences()(store.dispatch, store.getState),
+        getMyTeams()(store.dispatch, store.getState),
+        getMyTeamMembers()(store.dispatch, store.getState),
```

Comment:
nit: I think all of these can be done like in like 84, e.g. `store.dispatch(getMe())`. But no big deal either way.

Reply:
oh yeah, after years I am still confused by actions. I'll update.